config-example.js

```diff
@@ -10,6 +10,10 @@ export const config = {
   // Please note that (during testnet phrase) your address balance (in zkwasm server) should > 0;
   UserPrivateKey: "0x{PRIVATE_KEY}",
 
+  DispatcherContract: "0x25AA9Ec3CA462f5AEA7fbd83A207E29Df4691380",
+  DispatcherQueryrApi: "https://zkwasm.hyperoracle.io/td/",
```

Comment:
these 2 should be in the `constants.js`, because user doesn't need to care about it.